src/CSE.cpp

```diff
@@ -1134,11 +1134,16 @@ LOCAL INT cse3( INT argc, const char* argv[])
 		if (rv==1)			// if cul detected input error
 			rc = RCBAD;			// set our error code. sets errorlevel to 2 below.
 
-		// conditionally delete input records now to make max memory available
-		if (rv != 2)					// not if not end input file: addl statements may ADD to present input.
-			if ( !(Top.chAutoSize==C_NOYESCH_YES     // not if both autosizing and main-simulating requested,
-			 &&  Top.chSimulate==C_NOYESCH_YES ) )	 //   cuz input needed to re-setup for main sim run
-				cul( 4, NULL, NULL, NULL, NULL);      	// cul.cpp. Does not clear 'probed' basAncs. Duplicate call ok.
+#if 0
```

Comment:
Do we want to just delete this and leave a comment "Prior to version X, input records were deleted to save space, but prevented some information from being available after a run (see [PR 259](https://github.com/cse-sim/cse/pull/259/files))."

Reply:
I think it is easier to leave it this way for a bit to make sure it survives (?)

Reply:
Part of me feels like this shouldn't be necessary, but I can't exactly figure out what the best practice should be. @tanaya-bigladder, any thoughts?

Reply:
I've recently tended to just delete sections that I'm `#if 0`'ing out, especially if they've really been dead for a long time and I just haven't written the death certificate. Sometimes with a comment, or more often, with an elaboration in the commit message.